web_src/js/components/RepoActionView.vue

```diff
@@ -32,16 +32,16 @@
       <div class="action-view-left">
         <div class="job-group-section">
           <div class="job-brief-list">
-            <div class="job-brief-item" :class="parseInt(jobIndex) === index ? 'selected' : ''" v-for="(job, index) in run.jobs" :key="job.id" @mouseenter="onHoverRerunIndex = job.id" @mouseleave="onHoverRerunIndex = -1">
-              <a class="job-brief-link" :href="run.link+'/jobs/'+index">
+            <a class="job-brief-item" :href="run.link+'/jobs/'+index" :class="parseInt(jobIndex) === index ? 'selected' : ''" v-for="(job, index) in run.jobs" :key="job.id" @mouseenter="onHoverRerunIndex = job.id" @mouseleave="onHoverRerunIndex = -1">
+              <div class="job-brief-link">
```

Comment:
`div` inside `a` is not allowed, if I'm not mistaken.

Reply:
It's fine. This was forbidden in HTML 4, but has since been relaxed with HTML 5.